connect-inject/endpoints_controller_test.go

```diff
@@ -4656,93 +4656,78 @@ func TestCreateServiceRegistrations_withTransparentProxy(t *testing.T) {
 func TestRegisterServicesAndHealthCheck_skipsWhenDuplicateServiceFound(t *testing.T) {
 	t.Parallel()
 
-	cases := map[string]struct {
-		consulServiceMeta map[string]string
-	}{
-		"different k8s namespace meta": {
-			consulServiceMeta: map[string]string{MetaKeyKubeNS: "some-other-ns"},
-		},
-		"no k8s namespace meta": {
-			consulServiceMeta: nil,
-		},
```

Comment:
This diff is showing deleting this case and in-lining everything into the test since we only have one case now.

Reply:
Hooray deleting code :)